lib/ContractInteract/TokenRules.js

```diff
@@ -0,0 +1,180 @@
+'use strict';
+
```

Comment:
Add getter for `areDirectTransfersEnabled` and `token`

Reply:
Same as above!